Friendly-plans/app/src/main/java/matcher/RecyclerViewMatcher.java

```diff
@@ -0,0 +1,76 @@
+package matcher;
+
+import android.content.res.Resources;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+import org.hamcrest.TypeSafeMatcher;
+
+/**
+ * Created by dannyroa on 5/10/15.
+ */
```

Comment:
What do you think about changing it similar as in Validation class? I think it would be more clear that dannyroa is some good man who invented this matcher and share with the world :) URL + author?